plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/Range.java

```diff
@@ -0,0 +1,23 @@
+package io.snyk.languageserver.protocolextension.messageObjects.scanResults;
+
+public class Range {
```

Comment:
I think this class needs `equals` and `hashCode` methods to be able to sort & compare them